src/pages/evm/nfts/NftDetailsPage.vue

```diff
@@ -59,6 +59,10 @@ const ownerLink = computed(() => {
     return `${explorerUrl}/address/${nft.value.contractAddress}`;
 });
 
+const filteredAttributes = computed(() =>
+    nft.value?.attributes.filter(attr => attr.label !== undefined && attr.text !== undefined),
```

Comment:
```suggestion
    nft.value?.attributes.filter(attr => !!attr.label && !!attr.text),
```

suggesting because it may be that one of these is an empty string rather than undefined, in which case we still want to hide it